src/content/docs/dev/amplifier/its-hub/routing-mechanics.mdx

```diff
@@ -0,0 +1,171 @@
+payload = abi.encode(
+    MESSAGE_TYPE_RECEIVE_FROM_HUB,
+    sourceChain,       // True source chain
+    originalPayload    // Original ITS payload
+);
+
+gateway.callContract(destinationChain, destinationAddress, payload);
+```
+
+## Visualizing the Routing Process
+
+### ITS Chain A to ITS Chain B via ITS Hub
+
+<object
+  type="image/svg+xml"
+  data="/images/its-hub/its-hub-routing-dark.svg"
+  class="hidden dark:block w-full max-w-[60rem] mx-auto py-6"
+>
+  Your browser does not support SVG
+</object>
+<object
+  type="image/svg+xml"
+  data="/images/its-hub/its-hub-routing-light.svg"
+  class=" dark:hidden w-full max-w-[60rem] mx-auto py-6"
+>
+  Your browser does not support SVG
+</object>
+
+## Handling Different Token Types
+
+### ITS Tokens vs. Gateway Tokens
+
+- **ITS Tokens**: Managed entirely by the ITS protocol. The ITS Hub handles their deployment and transfer without special considerations.
+- **Gateway Tokens**: Minted and burned via the Axelar protocol. The ITS Hub interacts with the Axelar Gateway to handle these tokens, ensuring proper minting and burning.
+
+## Interacting with Different Chain Types
+
+### Amplifier Chains
+
+- **Smart Contract Chains**: Have ITS Edge Contracts similar to EVM chains. The routing mechanics are consistent with the dual GMP call approach.
+- **Non-Smart Contract Chains**: The Axelar Gateway on these chains acts as an ITS Edge Contract, translating ITS commands into custom token transfer or deployment transactions.
+
+**Amplifier Chain to ITS Hub**
+
+<object
+  type="image/svg+xml"
+  data="/images/its-hub/amplifier-to-its-dark.svg"
+  class="hidden dark:block w-full max-w-[60rem] mx-auto py-6"
+>
+  Your browser does not support SVG
+</object>
+<object
+  type="image/svg+xml"
+  data="/images/its-hub/amplifier-to-its-light.svg"
+  class=" dark:hidden w-full max-w-[60rem] mx-auto py-6"
+>
+  Your browser does not support SVG
+</object>
+
+**ITS Hub to Amplifier Chain**
+
+<object
+  type="image/svg+xml"
+  data="/images/its-hub/its-to-amplifier-dark.svg"
+  class="hidden dark:block w-full max-w-[60rem] mx-auto py-6"
+>
+  Your browser does not support SVG
+</object>
+<object
+  type="image/svg+xml"
+  data="/images/its-hub/its-to-amplifier-light.svg"
+  class=" dark:hidden w-full max-w-[60rem] mx-auto py-6"
+>
+  Your browser does not support SVG
+</object>
+
+### Consensus Chains
+
+- Use the existing Axelar protocol for GMP calls.
+- The ITS Hub interacts with consensus modules to handle GMP calls with tokens (Gateway tokens).
+
+**Consensus Chain to ITS Hub with Gateway Token**
+
+<object
+  type="image/svg+xml"
+  data="/images/its-hub/consensus-to-its-dark.svg"
+  class="hidden dark:block w-full max-w-[60rem] mx-auto py-6"
+>
+  Your browser does not support SVG
+</object>
+<object
+  type="image/svg+xml"
+  data="/images/its-hub/consensus-to-its-light.svg"
+  class=" dark:hidden w-full max-w-[60rem] mx-auto py-6"
+>
+  Your browser does not support SVG
```

Comment:
Are there browsers that don't support SVG? If it's less than 1%, let's leave these off.